app/models.py

```diff
@@ -37,10 +50,11 @@ def print_board(self):
         print(full_board)
 
     def is_valid_move(self, move: PieceMove) -> bool:
-        pass
+        return 1
 
     def make_move(self, move: PieceMove) -> bool:
```

Comment:
Let's correct the function's signature:
```suggestion
    def make_move(self, move: PieceMove):
```

Reply:
ofc!